hadoop-hdds/docs/content/interface/O3fs.md

```diff
@@ -59,10 +59,15 @@ Please add the following entry to the core-site.xml.
 </property>
 <property>
   <name>fs.defaultFS</name>
-  <value>o3fs://bucket.volume.OzoneServiceId</value>
+  <value>o3fs://bucket.volume</value>
 </property>
 ```
+<div class="alert alert-warning" role="alert">
+
+Tip: For the OM HA cluster, you need to specify the ozone service id.
```

Comment:
Add an example URL also for HA?

Reply:
Modify it like `Tip: For the [OM HA]({{< ref "feature/OM-HA.md">}}) cluster`？

Reply:
Ya that would be also good. But giving an example here would be a good idea. As this doc talks in general about O3fs.

**Example as below**
Tip: For the OM HA cluster, you need to specify the ozone service id. For example, if ozone.om.service.ids = ozone1, then the URL is o3fs://bucket.volume.ozone1

Reply:
Your suggestion is also good, it will be more intuitive. Then I will combine the two, what do you think?

Reply:
like this.
![image](https://user-images.githubusercontent.com/72794035/123412252-fa62bf00-d5e3-11eb-8009-9828403c64ca.png)

Reply:
Yes, looks good.

Reply:
Let me modify it.